src/components/Swap/Swap.tsx

```diff
@@ -148,6 +149,16 @@ export const Swap: React.FC<ISwap> = ({
 
   const timeoutRef = useRef<number>(0)
 
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    const tokenFromAddress = tokens[tokenFromIndex ?? -1]?.assetAddress
+    const tokenToAddress = tokens[tokenToIndex ?? -1]?.assetAddress
+    navigate(`/swap/${tokenFromAddress || '-'}/${tokenToAddress || '-'}`, {
```

Comment:
maybe it's better to add token names instead of addresses to URL, like we have on create position. It is more readable for user